go.sh

```diff
@@ -6,6 +6,10 @@ export LOG_UUID=FALSE
 
 . /defaults.sh
 
+if [ "${AWS_REGION}" == "" ]; then
+    exit_error_msg "AWS_REGION has not been specified, and is required."
+fi
+
```

Comment:
We only _need_ AWS_REGION if `DOWNLOAD_VIA_S3_VPC_ENDPOINT == TRUE` and we are already setting a default of an empty string. I'd be tempted to set the above default to "eu-west-1" and remove this check?